SU-SWS/saa_victoria_subtheme
# Adapt 1365  footer spacing

# READY FOR REVIEW

# Summary
- Remove margin following address paragraph, which contributes to spacing and creates too much / uneven spacing in fat footer.

# Review By (Date)
- 1/6/20

# Urgency
- How critical is this PR? low

# Steps to Test

1. Pull down the branch or check out:  https://saauserguide-test.sites.stanford.edu/site-building-guide/saa-components-stanford-sites/page-banners
2. Clear the site cache
3. Then test that the map link paragraph has margin 0 so that the spacing between it is the same and not bigger than the spacing above social media icons.
 this margin bottom in orange should be removed:

![image](https://user-images.githubusercontent.com/12898896/103589199-265c7d80-4ea8-11eb-9553-3a4b7dae6812.png)


# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules? only affects fat footer on saa sites

# Associated Issues and/or People
- JIRA ticket ADAPT-1365
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
